### PR TITLE
Added Field Case Updated tests

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

Field can submit an update to a case to change the expected CE capacity. If the new expected capacity is below the actual responses, it'll send a cancel message to field.

# What has changed

- Added new feature tests for Field Case Updated events

# How to test?

- Run with other branches

# Links

[Trello](https://trello.com/c/Stb2xLG5)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[RMC-339 Handle FieldCaseUpdated event (8)](https://trello.com/c/Stb2xLG5/651-rmc-339-handle-fieldcaseupdated-event-8)

File: acceptance_tests/features/field_case_updated.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    Given sample file "sample_1_english_CE_estab.csv" is loaded successfully
+    When a Field Case Updated message has been sent with expected capacity as "0"
+    Then a case_updated msg is emitted where "ceExpectedCapacity" is "0"
+    And the events logged for the case are [SAMPLE_LOADED,FIELD_CASE_UPDATED]
+    And a CANCEL action instruction is sent to field work management with address type "CE"
+
+  Scenario: A request to change the CE expected capacity for a case and a cancel message is not sent to field
+    Given sample file "sample_1_english_CE_estab.csv" is loaded successfully
+    When a Field Case Updated message has been sent with expected capacity as "3"
+    Then a case_updated msg is emitted where "ceExpectedCapacity" is "0"
```
Comment (liamtoozer) [inline acceptance_tests/features/field_case_updated.feature]: I think `ceExpectedCapacity` needs to be a 3?

**Review by liamtoozer — CHANGES_REQUESTED** [review]
The last scenario fails for me but I think it's because of the `ceExpectedCapacity` value?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**